inference-engine/tests/functional/plugin/shared/include/import_export_tests/import_export_base.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "functional_test_utils/layer_test_utils.hpp"
+
+#include <ie_core.hpp>
+
+typedef std::tuple<
+    InferenceEngine::Precision,         // Network Precision
+    std::string,                        // Target Device
+    std::map<std::string, std::string>, // Export Configuration
+    std::map<std::string, std::string>  // Import Configuration
+> exportImportNetworkParams;
+
+namespace LayerTestsDefinitions {
+
+class ImportNetworkTestBase : public testing::WithParamInterface<exportImportNetworkParams>,
```

Comment:
Looks like, we should move this class to the `funcTestUtils`

Reply:
Thanks! Done.

Reply:
Again, I said about the `funcTestUtils`